### PR TITLE
DQueue node

### PR DESCRIPTION
**Pull request by akashzcoder**
Dequeue node for LRU caching

### CODE REVIEW

File: pache/lru_cache.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function
+
+"""
+QNode -> holds key and value; as well as pointers to previous and next nodes.
+"""
+
+
+class DQNode(object):
```
Comment (Shabirmean) [inline pache/lru_cache.py]: How about `PacheItem`?

File: pache/lru_cache.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    def __init__(self, key: object, value: object):
+        """
+        Initialize the Dequeue Node.
```
Comment (Shabirmean) [inline pache/lru_cache.py]: Also, let's update the comment to reflect  the rename to `PacheItem`

File: pache/lru_cache.py (hunk 1 above)
Comment (Shabirmean) [reply pache/lru_cache.py]: If you rename let's update everywhere the term `DQNode` is used

File: pache/lru_cache.py (hunk 2 above)
Comment (akashzcoder) [reply pache/lru_cache.py]: Renamed the class. 0f3b0a69cd7334bdc1393dec0821b730f7889a57

**Pull request merged**